programs/compressed-token/src/process_transfer.rs

```diff
@@ -191,6 +191,14 @@ pub fn add_token_data_to_input_compressed_accounts(
     Ok(())
 }
 
+// TODO: consider moving this function to helpers
+/// Get static cpi signer seeds
+pub fn get_cpi_signer_seeds() -> [&'static [u8]; 2] {
+    let bump: &[u8; 1] = &[255];
+    let seeds: [&'static [u8]; 2] = [b"cpi_authority", bump];
```

Comment:
```suggestion
    let seeds: [&'static [u8]; 2] = [CPI_AUTHORITY_SEED, bump];
```

Reply:
will do in follow-up pr with all other occurrences